### PR TITLE
NM-59 rows view 🧐

### PR DESCRIPTION
**Pull request by lubomir1618**

### CODE REVIEW

File: src/components/teamwrapper/TeamWrapper.tsx [hunk 1]
```diff
@@ -7,57 +7,77 @@ import { Members } from '../members/Members';
...
-export interface ITeam {
-  name: String;
-  id: Number;
-  stats: TeamStatsRow[];
-  statsRings: TeamStatsRing[];
-  statType: StatType;
+export interface IProps {
+  name?: String;
+  id?: Number;
+  label?: string;
```
Comment (dullus) [inline src/components/teamwrapper/TeamWrapper.tsx]: no Ucase types!  string, number

File: src/components/teamwrapper/TeamWrapper.tsx [hunk 2]
```diff
@@ -70,26 +90,24 @@ export default class TeamWrapper extends React.Component<ITeam, any> {
...
     }
   }
 
   private renderRings(): JSX.Element {
     return (
       <div className={styles.graphs}>
-        {this.props.statsRings.map((ring, i) => (
-          <StatsRing key={i} {...ring} />
-        ))}
+        {this.props.statsRings ? this.props.statsRings.map((ring, i) => <StatsRing key={i} {...ring} />) : ''}
```
Comment (dullus) [inline src/components/teamwrapper/TeamWrapper.tsx]: Performance - no expensive operations inside JSX object. Set constant above return (..

File: src/components/teamwrapper/TeamWrapper.module.css [hunk 3]
```diff
@@ -50,3 +51,73 @@
...
   line-height: 12px;
 }
+
+.teamwrapper_list {
+  margin: 20px;
+  display: inline-block;
+  text-decoration: none;
+  color: #444444;
+  border: 1px solid #efefef;
+  font-family: Roboto;
```
[the remark this answers is not in the document]
Comment (lubomir1618) [reply src/components/teamwrapper/TeamWrapper.module.css]: done

File: src/components/teamwrapper/TeamWrapper.tsx (hunk 1 above)
Comment (lubomir1618) [reply src/components/teamwrapper/TeamWrapper.tsx]: done

File: src/components/teamwrapper/TeamWrapper.tsx (hunk 2 above)
Comment (lubomir1618) [reply src/components/teamwrapper/TeamWrapper.tsx]: done

**Pull request merged**